python/lihil/config.yaml

```
@@ -1,7 +1,7 @@
 framework:
-  website: www.lihil.cc/lihil/
+  website: www.lihil.cc
   github: raceychan/lihil
-  version: 0.2
+  version: 0.2.22
```

no need to be so precise 😛 

```suggestion
  version: 0.2
```

It is aligned with sematics on versionning here, using `X.Y` using `Z` will be time consuming (more than now) for maintenance

---

@waghanza would you be able to directly make changes on my commit? I do agree that we should keep it in the x.y format